tests/test_fileresolver.py

```diff
@@ -130,6 +131,7 @@ def test_paths_iterator(self) -> None:
 class CustomFileResolverTest(unittest.TestCase):
     """Tests for the CustomFileResolver class."""
 
+    @unittest.skipIf(platform.system() == 'Windows', "Skipping test on Windows due to path separator issues")
```

Comment:
I assume fixing this path separator issue will be a follow-up pull request?

Reply:
I'll change this, yes